.circleci/config.yml

```diff
@@ -186,18 +186,21 @@ commands:
           steps:
             - save_cache:
                 name: "save api gradle cache"
+                # See https://docs.gradle.org/current/userguide/directory_layout.html#dir:project_root
+                # Save Project-specific cache and Global cache (for everything that’s not project-specific).
                 paths:
+                  - .gradle
```

Comment:
Good find. Does this help? Did you confirm there are contents in this directory by SSHing into the Circle VM?

Reply:
I did `ls -al` in `api` dir. cmd outputs confirms `.gradle` dir exists after gradle job.